workspace/viewer/PdfJsViewer.ts

```diff
@@ -364,6 +362,14 @@ export class PdfJsViewer {
     }
   }
 
+  public showAll(url:string, isResize: boolean = false){
+    var pdfViewer = this;
+    pdfjs.getDocument(url).then(function(pdf){
+      for(var i = 1; i <= pdf.numPages; i++){
```

Comment:
In TypeScript world people use `let` instead of `var`. Also, please insert a space after `for`

Reply:
Done.